### ISSUE TITLE
Better understanding of the concept 

### ISSUE THREAD

**Issue by abhiagwl4262** [cvg/SOLD2#84]
So this work says that it handles data irrespective of whether annotations are there or not. 

If I want to guide line detection with the annotations, is that possible ?

**Comment by rpautrat** [comment]
Yes, you could train SOLD2 using existing human annotations. You just need to convert your annotations to the SOLD2 format (which I think you have by now). Then you can copy the structure of the holicity dataloader for your dataset.

I am not sure what kind of lines are present in aerial images, but SOLD2 requires lines that are often connected together (like a wireframe), since it first detects the junctions, then connect them with a line. It can detect isolated lines with 2 junctions of course, but it might work better when several lines are connected together.
